ts/store/reducers/authentication.ts

```diff
@@ -125,7 +141,8 @@ const reducer = (
       ...state,
       ...{
         kind: "LoggedOutWithIdp",
-        idp: action.payload
+        idp: action.payload,
+        isSessionExpired: false
```

Comment:
you don't need to set `isSessionExpired` (or `reason`) again, as it will come from `...state`